Mathlib/RingTheory/TwoSidedIdeal/Basic.lean

```diff
@@ -156,17 +156,16 @@ def mk' (carrier : Set R)
         rw [show a + c - (b + d) = (a - b) + (c - d) by abel]
         exact add_mem h1 h2 }
 
-lemma mem_mk' (carrier : Set R)
-    (zero_mem : 0 ∈ carrier)
-    (add_mem : ∀ {x y}, x ∈ carrier → y ∈ carrier → x + y ∈ carrier)
-    (neg_mem : ∀ {x}, x ∈ carrier → -x ∈ carrier)
-    (mul_mem_left : ∀ {x y}, y ∈ carrier → x * y ∈ carrier)
-    (mul_mem_right : ∀ {x y}, x ∈ carrier → x * y ∈ carrier)
-    (x : R) :
-    x ∈ mk' carrier zero_mem add_mem neg_mem mul_mem_left mul_mem_right ↔ x ∈ carrier := by
+lemma mem_mk' (carrier : Set R) (h₁ h₂ h₃ h₄ h₅) (x : R) :
```

Comment:
Should the `mem_blah` lemmas be `@[simp]`?

Reply:
This seems like a good one to be simp